beaker.gemspec

```diff
@@ -47,7 +47,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'beaker-hostgenerator'
 
   # Optional provisioner specific support
-  s.add_runtime_dependency 'rbvmomi', '~> 1.8'
+  s.add_runtime_dependency 'rbvmomi', '= 1.8'
```

Comment:
I think you need to make this [1.8.2](https://rubygems.org/gems/rbvmomi/versions/1.8.2).